File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        
        for (int i = 0; i < N; i++) {
            String seminar = sc.next();
            
            if (seminar.equals("Algorithm")) {
                System.out.println(204);
            } else if (seminar.equals("DataAnalysis")) {
                System.out.println(207);
            } else if (seminar.equals("ArtificialIntelligence")) {
                System.out.println(302);
            } else if (seminar.equals("CyberSecurity")) {
                System.out.println("B101");
            } else if (seminar.equals("Network")) {
                System.out.println(303);
            } else if (seminar.equals("Startup")) {
                System.out.println(501);
            } else {
                System.out.println(105);
            }
        }
    }
}

File: 백준/Bronze/30087. 진흥원 세미나/진흥원 세미나.cs
using System;

class Q30087
{
    static void Main(string[] args)
    {
        int N = int.Parse(Console.ReadLine());
        
        for (int i = 0; i < N; i++)
        {
            String seminar = Console.ReadLine();
            
            if (seminar.Equals("Algorithm"))
            {
                Console.WriteLine(204);
            }
            else if (seminar.Equals("DataAnalysis"))
            {
                Console.WriteLine(207);
            }
            else if (seminar.Equals("ArtificialIntelligence"))
            {
                Console.WriteLine(302);
            }
            else if (seminar.Equals("CyberSecurity"))
            {
                Console.WriteLine("B101");
            }
            else if (seminar.Equals("Network"))
            {
                Console.WriteLine(303);
            }
            else if (seminar.Equals("Startup"))
            {
                Console.WriteLine(501);
            }
            else
            {
                Console.WriteLine(105);
            }
        }
    }
}